uwufetch.c

```diff
@@ -103,10 +105,8 @@ int main(int argc, char *argv[]) {
 				return 0;
 			case 'i':
 				ascii_image_flag = 1;
-				char *image_dir = NULL;
-				if(!optarg && argv[optind] != NULL && argv[optind][0] != '-') image_dir = argv[optind++];
-				else image_dir = optarg;
-				if(image_dir != NULL)  sprintf(image_name, "%s", image_dir);
+				if(!optarg && argv[optind] != NULL && argv[optind][0] != '-') sprintf(image_name, "%s", argv[optind++]);
```

Comment:
Just to make it clear, we need line 108 and line 109 to accept the option as `--image path/to/image` since `getopt_long` and `getopt` doesn't allow a space in optional arguments.